vsintegration/ProjectTemplates/TutorialProject/Template/Tutorial.fsx

```diff
@@ -370,16 +476,217 @@ module UnionTypes =
     /// This example prints all the cards in a playing deck.
     let printAllCards() = 
         for card in fullDeck do 
-            printfn "%s" (showCard card)
+            printfn "%s" (showPlayingCard card)
+
+    // Single-case DUs are often used for domain modeling.  This can buy you extra type safety
+    // over primitive types such as strings and ints.
```

Comment:
...as they cannot be implicitly cast to/from the type they wrap. For example, a function that takes a parameter of type Address will not accept a value of type string (and vice versa).